lms/views/api/d2l/files.py

```diff
@@ -12,17 +11,6 @@
 )
 
 
-@view_config(
```

Comment:
Again not a big improvement, we are replacing two view that are both one liners with another one liner.

The benefits, line count wise would be more evident in more complex views.